API.md

```diff
@@ -298,6 +298,85 @@ Status description (`statusText`): (see Operator-Service for full status list)
 | 60     | Publishing results            |
 | 70     | Job completed                 |
 
+
+## Initialise a compute service
+
+### POST /api/services/initializeCompute
+
+Somewhat similar to the compute endpoint, except datasets are all in the same key, no differentiation between a "main" dataset and additional ones.
```

Comment:
Can we rephrase this and refer to the scope of this endpoint which is to retrieve provider fees for a compute service?